{
  try {
    mAdapter.handleFailedOutgoingCall(callId,errorMessage);
  }
 catch (  RemoteException e) {
  }
}
